### ISSUE TITLE
Server integration with appium-windows-driver

### ISSUE THREAD

**Issue by mykola-mokhnach** [licanhua/YWinAppDriver#33]
Hello @licanhua 

Thank you for your great work on this project. My name is Mykola and I am one of Appium core developers. We are considering to incorporate your implementation of the WDA server with [Appium](https://github.com/appium/appium-windows-driver) instead of the one which has been abandoned by Microsoft. 

I would like to know if you are interested in such cooperation and if you have plans for further long time project maintenance and development. 

Basically, what we have on our roadmap for now is the full W3C protocol support and various bugfixes.

Thank you for your time.

**Comment by anunay1** [comment]
Good initiative 👍

**Comment by licanhua** [comment]
@mykola-mokhnach Good to know on it. I don't have long term plan on it. I didn't work on native app anymore, so I may put limited time on the maintains. If anyone is willing to put more time on the future planning and user engagement, I can add them to the admin

**Comment by mykola-mokhnach** [comment]
Ok, no problem. 

Let me ask then at the forum if someone would like to support/maintain the project.

**Comment by namilkimfree** [comment]
Hello @mykola-mokhnach 

I would like to test the W3C specs, and I have a couple of questions:

Can I directly connect to YWinAppDriver using Appium .NET and send requests to verify the behavior?
Or is it necessary to use appium-windows-driver for testing instead?
If I need to use appium-windows-driver, could you provide guidance on how to connect it locally to YWinAppDriver and run the tests?

Additionally, could you confirm if the existing JWP (JSON Wire Protocol) is also supported along with W3C?

Thank you.

**Comment by mykola-mokhnach** [comment]
@namilkimfree Unfortunately I did not try this driver myself.

Based on its readme it should be possible to setup a direct communication with the dotnet client.

As for appium-windows-driver, it would probably require some updates in the source code. We'd need to get rid of  JWP->W3C transformation patches required by the original WAD first.

**Comment by licanhua** [comment]
@namilkimfree It appears that official support for WinAppDriver has been discontinued for several years. A potential replacement for it is [power-automate](https://learn.microsoft.com/en-us/power-automate/getting-started). If you don't need the paid features, it is [free](https://mspowerautomate.com/is-power-automate-free-microsoft-power-automate-licensing/) to start with.

Attempting to bridge the gap between W3C and WinAppDriver may not be worth your time. My suggestion is to either continue using the old version of Appium, which doesn't require W3C, or switch to another solution.

**Comment by daluu** [comment]
For another alternative, if this one never gets update integration with Appium, then this one might be more promising: https://github.com/FlaUI/FlaUI.WebDriver

**Comment by namilkimfree** [comment]
@licanhua  @daluu 

Thanks for the replies
